src/tracks/ui/TrackView.cpp

```diff
@@ -204,18 +225,24 @@ struct TrackPositioner final : ClientData::Base, wxEvtHandler
    {
       e.Skip();
 
-      auto iter =
-         TrackList::Get( mProject ).Find( e.mpTrack.lock().get() );
-      if ( !*iter )
-         return;
+      auto& tracks = TrackList::Get(mProject);
 
-      auto prev = iter;
-      auto yy = TrackView::GetCumulativeHeight( *--prev );
+      auto iter = tracks.Find( e.mpTrack.lock().get() );
+      if (iter == tracks.end())
+         return;
 
+      auto yy = 0;
+      if (iter == tracks.begin())
+          yy = kTopMargin;
+      else
+          yy = TrackView::Get(**iter.advance(-1)).GetBottom() + kSeparatorThickness;
```

Comment:
Is this correct or off by one?

Reply:
It's same as in drawing routine I suppose